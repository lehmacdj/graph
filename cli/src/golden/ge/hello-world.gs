mk hello
cd hello
mk world/"!"
cd world/"!"
cd 0:#
ls
