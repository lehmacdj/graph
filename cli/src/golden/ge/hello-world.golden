==========stdout==========
g> g> g> g> g> g> "hello" at 1
g> Goodbye!
==========stderr==========
==========graph-dump==========
0{in=[], out=[1 via "hello"]}
1{in=[0 via "hello"], out=[2 via "world"]}
2{in=[1 via "world"], out=[3 via "!"]}
3{in=[2 via "!"], out=[]}
